src/components/dropdown/CustomRender.tsx

```diff
@@ -188,7 +189,12 @@ export const DropdownList = ({selectProps, ...props}: any) => {
       }
     }, [select?.state?.focusedOption]);
   }
-  return <components.MenuList {...props}>{props.children}</components.MenuList>;
+  return <components.MenuList
+    className={ 'ui-toolkit__dropdown__menuList' }
```

Comment:
I think the styling for this component should be in ui-toolkit-components because styles knows nothing about components.MenuList

Reply:
Where is this class coming? I di not see it on the styles pr

Reply:
Removed before PR was public, will resolve conversation.